relnotes/mirror.feature.md

```diff
@@ -20,3 +20,21 @@ Notes:
   expected to have any effect on applications, but is a noteworthy change in
   behaviour.
 
+### Extensible channel mirror works with new channel mirror class
+
+`dhtproto.client.legacy.internal.helper.ExtensibleChannelMirror`
+
+The new template -- `ExtensibleMirror` -- allows the extensible mirror
+functionality to be used with the new `Mirror` class. Usage like:
+```D
+alias ExtensibleChannelMirror!(SchedulingDhtClient,
+    RawRecordDeserializer!(Record), DeserializedRecordCache!(Record))
+    ExampleMirror;
+```
+should be changed to
+```D
+alias ExtensibleMirror!(SchedulingDhtClient, Mirror!(SchedulingDhtClient),
+    RawRecordDeserializer!(Record), DeserializedRecordCache!(Record))
+    ExampleMirror;
```

Comment:
Is this change worth deprecation of `ExtensibleChannelMirror`?

Reply:
Well, usage is intended to be entirely replaced, gradually. I get the feeling that a deprecation may be a _bit_ too much, at this stage, though, as that implies to many people "stop using this code ASAP", which is not quite the case here. We need to be careful, as the new code may of course contain new bugs.

Reply:
Yes, I share the same feeling.

Reply:
Tempting as it is to be gung ho about it and kill off the old one ;)